main.go

```diff
@@ -19,6 +19,8 @@ func main() {
 		"port of Contrail Controller API")
 	var logLevelString = flag.String("logLevel", "Info",
 		"log verbosity (possible values: Debug|Info|Warn|Error|Fatal|Panic)")
+	var vswitchName = flag.String("vswitchName", "Layered Ethernet0",
```

Comment:
In May 2017 update of Windows Server vswitch names created alongside HNS networks are named `Layered_<ifname>`. Maybe we could build `vswitchName` using provided interface name?